docs/source/usage/tbg.rst

```diff
@@ -3,15 +3,29 @@
 TBG
 ===
 
-.. sectionauthor:: Axel Huebl, Richard Pausch
+.. sectionauthor:: Axel Huebl
 .. moduleauthor:: René Widera
 
-todo: explain idea and use case
+Our tool *template batch generator* (``tbg``) abstracts program runtime options from technical details of supercomputers.
+On a desktop PC, one can just execute a command interactively and instantaneously.
+Contrarily on a supercomputer, resources need to be shared between different users efficiently via *job scheduling*.
+Scheduling on today's supercomputers is usually done via *batch systems* that define various queues of resources.
+
+An unfortunate aspect about batch systems from a user's perspective is, that their usage varies a lot.
+And naturally, different systems have different resources in queues that need to be described.
+
+We abstract the description of queues, resource acquisition and job submission away from PIConGPU user input via *template files* (``.tpl``).
+For example, the ``.cfg`` file defines how many *devices* shall be used for computation, but the ``.tpl`` file calculates how many *physical nodes* will be requested.
```

Comment:
I think here `the ` .cfg file is rather unnatural, we are talking in general about a configuration file, right?